bokeh/models/sources.py

```diff
@@ -256,6 +256,14 @@ def _check_column_lengths(self):
         if len(lengths) > 1:
             return str(self)
 
+
+class GeoJSONDataSource(DataSource):
+    geojson = JSON(help="""
```

Comment:
Yes, that's listed as a ToDo, and it'll be a small one, I'm trying to get things through in incremental steps and not have mammoth PRs.